tests/server/models/wine-test.js

```diff
@@ -0,0 +1,122 @@
+var dbURI = 'mongodb://localhost:27017/testingDB';
+var clearDB = require('mocha-mongoose')(dbURI);
+
+var sinon = require('sinon');
+var expect = require('chai').expect;
+var mongoose = require('mongoose');
+var Promise = require('bluebird')
+
+// Require in all models.
+require('../../../server/db/models');
+
+var Wine = mongoose.model('Wine');
+var Review = mongoose.model('Review');
+var User = mongoose.model('User');
+
+describe('Wine model', function () {
+
+    beforeEach('Establish DB connection', function (done) {
+        if (mongoose.connection.db) return done();
+        mongoose.connect(dbURI, done);
+    });
+
+    afterEach('Clear test database', function (done) {
+        clearDB(done);
+    });
+
+    it('should exist', function () {
+        expect(Wine).to.be.a('function');
+    });
+
+    describe('Wines', function () {
+
+        
+            var user = new User({
+                email: 'GHA@gmail.com'
+            })
+
+            var wine = new Wine({
+                type: "red",
+                variety: "Malbec",
+                region: "Spain",
+                winery: "L&P",
+                year: 2016,
+                price: 10
+            });
+
+            var review1 = new Review({
+                stars: 3,
+                wine: wine._id,
+                author: user._id
+            });
+
+            var review2 = new Review({
+                stars: 5,
+                wine: wine._id,
+                author: user._id
+            });
```

Comment:
At this point, let's say yes, but we added an issue to the backlog to see if we want prevent a user from creating a new review. Maybe they can only edit their own reviews?